python/whylogs/api/logger/experimental/logger/actor/thread_rolling_logger.py

```diff
@@ -350,25 +373,31 @@ def _process_track_message(self, message: TrackMessage) -> None:
             timestamp_ms = message.timestamp_ms
             data = message.data
 
-            ts = timestamp_ms or current_time_ms()
+            ts = timestamp_ms or self.current_time_ms()
             dataset_timestamp = truncate_time_ms(ts, self._aggregate_by)
             profile_container = self._get_profile_container(dataset_timestamp)
+            # TODO consider porting out the aggregation logic that the process rolling logger uses to batch up
+            # data and minimize the amount of track calls. It makes a really big difference.
             profile_container.track(data)
             if message.result is not None:
                 message.result.set_result(None)
         except Exception as e:
             if message.result is not None:
                 message.result.set_exception(e)
 
-    def _status(self) -> LoggerStatus:
-        result: Future[LoggerStatus] = Future()
+    def status(self, timeout: Optional[float] = 1.0) -> LoggerStatus:
```

Comment:
consider defining a few constants for the default timeouts so its easier to change these later if we have to, I see there are a few other places that use 0.1 for example.